docs/EmbeddedAppIOS.md

```diff
@@ -15,7 +15,8 @@ Since React makes no assumptions about the rest of your technology stack – it
 - [Node.js](http://nodejs.org)
   - Install **nvm** with [its setup instructions here](https://github.com/creationix/nvm#installation). Then run `nvm install node && nvm alias default node`, which installs the latest version of Node.js and sets up your terminal so you can run it by typing `node`.  With nvm you can install multiple versions of Node.js and easily switch between them.
   - If you are using Node 5.0 or newer, we recommend installing npm 2, which is much faster than npm 3. After installing Node, run `npm install -g npm@2`
-- Install your copy of React Native under your `node_modules` directory where your JS resides.
+- Install the `react-native` node package in a subdirectory of your project.  It will be helpful, when we're bundling and serving js files with the React Native development server (see below), to have this in a directory by itself, with no other child or sibling js files.  We suggest running the install command from the root directory of your project (so that node_modules ends up as sibling to your .xcodeproj file):
```

Comment:
Done.

On Sun, Dec 27, 2015 at 12:48 AM, Satyajit Sahoo <notifications@github.com>
wrote:

> In docs/EmbeddedAppIOS.md
> <https://github.com/facebook/react-native/pull/4828#discussion_r48452797>:
>
> > @@ -15,7 +15,8 @@ Since React makes no assumptions about the rest of your technology stack – it
> >  - [Node.js](http://nodejs.org)
> >    - Install **nvm** with [its setup instructions here](https://github.com/creationix/nvm#installation). Then run `nvm install node && nvm alias default node`, which installs the latest version of Node.js and sets up your terminal so you can run it by typing `node`.  With nvm you can install multiple versions of Node.js and easily switch between them.
> >    - If you are using Node 5.0 or newer, we recommend installing npm 2, which is much faster than npm 3. After installing Node, run `npm install -g npm@2`
> > -- Install your copy of React Native under your `node_modules` directory where your JS resides.
> > +- Install the `react-native` node package in a subdirectory of your project.  It will be helpful, when we're bundling and serving js files with the React Native development server (see below), to have this in a directory by itself, with no other child or sibling js files.  We suggest running the install command from the root directory of your project (so that node_modules ends up as sibling to your .xcodeproj file):
>
> Sorry for late reply. But seems you forgot to change this one,
>
> Install the react-native node package in a subdirectory of your project. It will be helpful, when we're bundling and serving js files with the React Native development server (see below), to have this in a directory by itself, with no other child or sibling js files. We suggest running the install command from the root directory of your project (so that node_modules ends up as sibling to your .xcodeproj file):
>
> to
>
> Install the `react-native` package from npm by running the following command in the root directory of your project:
>
>     npm install react-native
>
> This will install React Native under a directory named `node_modules`.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/react-native/pull/4828/files#r48452797>.
>